server/util/ServerCommand.java

```diff
@@ -49,6 +49,14 @@ default ImportData asImportData() {
         throw GraknException.of(ILLEGAL_CAST);
     }
 
+    default boolean isExportData() {
+        return false;
+    }
+
+    default ExportData asExportData() {
+        throw GraknException.of(ILLEGAL_CAST);
```

Comment:
doesn't our ILLEGAL_CAST take parameters saying which class was being cast to which other class?

Reply:
Yep I missed that, will add.